provider/openstack/local_test.go

```diff
@@ -1271,7 +1273,30 @@ func (s *localServerSuite) TestEnsureGroup(c *gc.C) {
 	c.Assert(group.Name, gc.Equals, "test group")
 	assertRule(group)
```

Comment:
Existing issue, but assert rule really needs to ensure that the expected egress rules are there, not just skip over them with no checks